### ISSUE TITLE
Support for Fujifilm X-T5

### ISSUE THREAD

**Issue by sjones1011** [timelapseplus/VIEW#47]
Hi Elijah,

Thanks so much for adding the Fujifilm X-H2 & X-H2s!!   Any chance of adding the X-T5 shortly?  Hoping it's virtually identical to the other two and can come onboard quickly.

Thanks for your consideration on this!

**Comment by elijahparker** [comment]
Definitely! I’m thinking of getting the X-T5 myself as a compliment to the
X-H2S for landscape. Do you know the USB product ID for the X-T5?

**Comment by sjones1011** [comment]
<!--
/* Font Definitions */
@font-face
	{font-family:"Cambria Math";
	panose-1:2 4 5 3 5 4 6 3 2 4;}
@font-face
	{font-family:Calibri;
	panose-1:2 15 5 2 2 2 4 3 2 4;}
/* Style Definitions */
p.MsoNormal, li.MsoNormal, div.MsoNormal
	{margin:0in;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
a:link, span.MsoHyperlink
	{mso-style-priority:99;
	color:blue;
	text-decoration:underline;}
.MsoChpDefault
	{mso-style-type:export-only;}
@page WordSection1
	{size:8.5in 11.0in;
	margin:1.0in 1.0in 1.0in 1.0in;}
div.WordSection1
	{page:WordSection1;}

**Comment by stonecoldcoffeephotography** [comment]
do you know where to purchase the correct USB to USB-C cable for canon? the normal USB from the view to USB-C to camera doesnt seem to work.
